testing/playwright/tests/css-regression.spec.ts

```
@@ -47,35 +47,47 @@ adminTest.describe('CSS regression', () => {
-    if (csrfToken2) {
+      // Navigate to another page and check html[data-theme]
+      await page.goto('subnets.php');
+      const theme = await page.locator('html').getAttribute('data-theme');
+      expect(theme).toBe('dark');
+    } finally {
+      // Always restore the theme to auto so a persisted theme=dark cannot
+      // leak into later tests. If the restore CSRF token is unavailable the
+      // restore cannot be performed — fail loudly rather than leak state.
+      const csrfToken2 = await page.locator('meta[name="ipam-csrf"]').getAttribute('content').catch(() => '');
+      if (!csrfToken2) {
+        throw new Error('could not get CSRF token to restore theme=auto — persisted theme would leak');
```

Biome (2.4.15)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)